src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -255,18 +255,6 @@ internal override void ForceComplete(SourceLocation locationOpt, CancellationTok
                         _state.SpinWaitComplete(CompletionPart.FinishValidatingReferencedAssemblies, cancellationToken);
                         break;
 
-                    case CompletionPart.StartMemberChecks:
```

Comment:
>case [](start = 20, length = 4)

I didn't understand why this could be removed. Aren't we losing some diagnostics?

Reply:
We were reporting those diagnostics before, but now emitting of this attribute is tied to the other attributes so any diagnostics will be reported from those attributes.

---
In reply to: [301355338](https://github.com/dotnet/roslyn/pull/37019#discussion_r301355338) [](ancestors = 301355338)